README.md

```diff
@@ -43,4 +54,14 @@ If you are using the Dwarf wifi, the `DWARF_IP` is 192.168.88.1. If you are usin
 python server.py
 ```
 
-6. Select an object in Stellarium, and issue a slew command. The Dwarf II should move to that object.
+On Windows : 
+
+```
+python3 server.py
+```
+
+6. Start the dwarf II and use the mobile app and go to Astro Mode and do the calibration
+
+7. Select an object in Stellarium, and issue a slew command. (shortcut : Alt + 1). The Dwarf II should move to that object.
```

Comment:
This app also work on Macs. Shortcut on Macs is Command + number.  

The number is not always 1. People need to look at the Stellarium telescope plugin to figure out which number to use.